medallion/test/data/default_data.json

```diff
@@ -158,7 +158,7 @@
                         "name": "Malicious site hosting downloader",
                         "pattern": "[url:value = 'http://z4z10farb.cn/4712']",
                         "pattern_type": "stix",
-                        "spec_version": "2.0",
```

Comment:
This property can be removed completely. This will better represent a 2.0 object.